src/content/docs/jellyfin-integration/configuring-shokofin.mdx

```diff
@@ -0,0 +1,237 @@
+
+Specials placement within seasons:
+
+    - Determines how Specials are placed within Seasons. Warning: Modifying this setting requires a recreation (read as; delete existing then create a new) of any libraries using this plugin — otherwise you will have mixed metadata.
+
+Mark specials
+
+    - Add a number to the title of each Special episode
+
+## User Settings
+
+Configure settings for:
+
+    - Select a user to add, modify or delete the user settings for.
+
+Enable synchronization features
+
+    - Enabling this setting is mandatory to enable the sync features below. Select the sync feature you would like to use by enabling the checkbox.
+
+Sync watch-state on import or refresh
+
+    - Enabling this setting will import the watch-state for items from Shoko on import or refresh.
+
+Sync watch-state after playback
+
+    - Enabling this setting will sync-back the watch-state to Shoko after playback has ended.
+
+Sync watch-state events during playback
+
+    - Enabling this setting will sync-back the watch-state to Shoko on every play/pause/resume/stop event during playback.
+
+Sync watch-state live during playback
+
+    - Enabling this setting will sync-back the watch-state to Shoko live during playback.
+
+Sync watch-state for restricted videos
+
+    - Enabling this setting will sync-back the watch-state to Shoko for restricted videos (H).
+
+Username:
+
+    - The username of the account to synchronize with the currently selected user.
+
+Password:
+
+    - The password for account. It can be empty.
+
+## Tag Settings
+
+Hide unverified tags.
+
+Hide art style related tags
+
+Hide misc info tags that may be useful
+
+Hide potentially plot-spoiling tags
+
+Hide any miscellaneous tags
+
+Hide any tags related to the setting — a time or place in which the story occurs.
+
+Hide any tags that involve how or where it aired, or any awards it got
+
+## Advanced Settings
+
+Enable Crash Reporting.
+
+    - Improve the plugin's stability by allowing Sentry to report any crashes, which helps us improve the plugin's stability. This may include paths to your videos involved in the operation, which some users may consider sensitive. We assure you that this data is used solely for diagnostic purposes. Participation is entirely optional but greatly appreciated.
+
+Public Shoko host URL:
+
+    - This is the public URL leading to where Shoko is running. It is used to redirect to Shoko if you click on a Shoko ID in the UI if Shoko and/or Jellyfin is running within a container. It should include both the protocol and the IP/DNS name.
+
+Ignored file extensions:
```

Comment:
ignored file extensions are also gone simce we're now using jf's built in inclusion list instead.